src/infrastructures/AudioRenderer.ts

```diff
@@ -878,7 +876,131 @@ export class ChannelStrip {
     audioContext: BaseAudioContext,
     options: ChannelStripOptions = { volume: 0.1 }
   ) {
-    this.gainNode = audioContext.createGain();
+    this.gainNode = new GainNode(audioContext);
     this.gainNode.gain.value = options.volume;
   }
 }
+
+export type LimiterOptions = {
+  readonly inputGain: number;
+  readonly outputGain: number;
+  readonly release: number;
+};
+
+/**
+ * リミッターです。大きい音を抑えます。
+ */
+export class Limiter {
+  private readonly inputGainNode: GainNode;
+  private readonly compNode: DynamicsCompressorNode;
+  private readonly correctionGainNode: GainNode;
+  private readonly outputGainNode: GainNode;
+
+  get input(): AudioNode {
+    return this.inputGainNode;
+  }
+
+  get output(): AudioNode {
+    return this.outputGainNode;
+  }
+
+  /**
+   * 入力ゲイン（dB）
+   */
+  get inputGain() {
+    return this.getGainInDecibels(this.inputGainNode);
+  }
+  set inputGain(value: number) {
+    this.setGainInDecibels(value, this.inputGainNode);
+  }
+
+  /**
+   * 出力ゲイン（dB）
+   */
+  get outputGain() {
+    return this.getGainInDecibels(this.outputGainNode);
+  }
+  set outputGain(value: number) {
+    this.setGainInDecibels(value, this.outputGainNode);
+  }
+
+  get release() {
+    return this.compNode.release.value;
+  }
+  set release(value: number) {
+    this.compNode.release.value = value;
+  }
+
+  get reduction() {
+    return this.compNode.reduction;
+  }
+
+  constructor(
+    audioContext: BaseAudioContext,
+    options: LimiterOptions = { inputGain: 0, outputGain: 0, release: 0.25 }
+  ) {
+    this.inputGainNode = new GainNode(audioContext);
+    this.compNode = new DynamicsCompressorNode(audioContext);
+    this.correctionGainNode = new GainNode(audioContext);
+    this.outputGainNode = new GainNode(audioContext);
+
+    this.compNode.threshold.value = -5;
+    this.compNode.ratio.value = 20;
+    this.compNode.knee.value = 8;
+    this.compNode.attack.value = 0;
+    this.compNode.release.value = options.release;
+    this.correctionGainNode.gain.value = 0.85;
```

Comment:
このあたりの値、どのような願いを込められて決定したのかをコメントしておくと将来のコミッターの助けになるかもです。（主に僕たちが忘れたときに･･･。）
コメント粒度の加減がちょっと掴めてないのですが、こんな感じなのかなというのを書いてみたので温度感の参考になれば！

```suggestion
    this.compNode.threshold.value = -5; // なんとなく
    this.compNode.ratio.value = 20; // 最大値
    this.compNode.knee.value = 8; // 0でも良いかも
    this.compNode.attack.value = 0; // 音割れ防止のため
    this.compNode.release.value = options.release;
    this.correctionGainNode.gain.value = 0.85; // なんとなく
```

Reply:
ありがとうございます！
コメントを追加しました！